src/molecules/modal/Modal.html

```diff
@@ -0,0 +1,64 @@
+<transition :name="modalMainTransition">
+  <div
+    v-show="visibility"
+    :aria-hidden="ariaHidden"
+    :aria-labelledby="ariaLabelledby"
+    :aria-describedby="ariaDescribedby"
+    class="a-modal"
+    role="dialog"
+  >
+    <div
+      @click.self="handleBackgroundClick"
+      class="a-modal__mask"
+    ></div>
+    <transition :name="modalContentTransition">
+      <div
+        class="a-modal__content"
+        v-show="visibility"
+        ref="modal"
+        role="document"
+        tabindex="0"
+        @keydown="handleKeydown"
+      >
+        <div
+          v-if="heading || closeButton"
+          class="a-modal__header"
+        >
+          <!-- @slot Custom heading (Scoped slot) -->
+          <slot
+            name="heading"
+            :modal-heading="heading"
+          >
+            <a-heading
+              v-if="heading"
+              :level="2"
+              class="a-modal__heading"
+            >
+              {{ heading }}
+            </a-heading>
+            <a-divider
+              v-if="heading"
+              class="a-modal__divider"
+            />
+          </slot>
+          <button
+            @click="hide(name)"
+            class="button button--icon a-modal__close-button"
+            type="button"
+          >
```

Comment:
Please add aria-label - you can see a violation in accessibility tab after modal open.
Probably you can use `<a-button>` with `<a-icon>`and update styles (cursor pointer is also missing)

Reply:
Aria-label is still missing :P